tests/test_compare_with_default_qubit.py

```diff
@@ -134,7 +134,11 @@ def circuit():
                         operation_class(*operation_pars, wires=operation_wires)
                         return qml.expval(observable_class(*observable_pars, wires=observable_wires))
 
-                    output = circuit()
+                    try:
+                        output = circuit()
+                    except IgnoreOperationException:
+                        continue
```

Comment:
Ideally, an ignored operation should not fail the tests! I would use `pytest.skip` here, except this test suite is so old, it predates our move to pytest and still includes an explicit loop over all operations :)

Reply:
Wow! Totally fine i'd say. It's amazing this old plugin still works.